README.md

```diff
@@ -11,22 +11,22 @@
 
 ## Introduction
 
-This Apprenticeship Program is looking for improve the iOS Technical skills. 
-When the Wizeliner ends this program the expected level would be two or above according the current necessities.
+This iOS Bootcamp is looking to improve your iOS Technical skills. 
+When you complete this program, the expected level would be two or above according the current necessities.
```

Comment:
I'm not sure if L2 is what we are expecting, I think is a solid L3, but we could double-check this with Sam. 
I understand that the readme had L2 because that was the expectation for the apprenticeship 2020